pkg/config/dynamicconf.go

```diff
@@ -5,17 +5,16 @@ import (
 	"fmt"
 	"reflect"
 
-	"github.com/jinzhu/configor"
 	"gorm.io/gorm"
 )
 
 // DynamicConfig model is a set of columns describing the dynamicConfig of the scoring engine
 type DynamicConfig struct {
 	ID uint64 `json:"id,omitempty"`
 	// Describes how long each round unit takes to execute in seconds. This value shuold have a minimum value enforced (something like 20 seconds)
-	RoundDuration uint64 `json:"round_duration,omitempty" default:"60"`
+	RoundDuration uint64 `json:"round_duration,omitempty"`
 	// Enables or disables competition globally
-	Enabled *bool `json:"enabled,omitempty" default:"false" gorm:"not null;default:false"`
+	Enabled *bool `json:"enabled,omitempty" gorm:"not null;default:false"`
```

Comment:
I can't find the default values set for RoundDuration in root of viper

Reply:
Adding that now. It took me a while to understand that the application injest both static and dynamic at start and uploads the dynamic config

Reply:
Yeah, that part might be confusing 😅. But yes, Dynamic config gets uploaded at the start of the app, if it isn't uploaded already. And if it is already there, then we just pull latest dynamic config. Dynamic config syncs its settings over DB, and the values are able to be manipulated, and synced across all instances

Reply:
sounds like something etcd would be a better solution for. viper evens support etcd connection